core/src/main/scala/org/broadinstitute/dsde/rawls/model/CloudPlatform.scala

```diff
@@ -18,6 +18,7 @@ object CloudPlatform extends Enumeration {
   def apply(platform: WSMCloudPlatform): CloudPlatform = platform match {
     case WSMCloudPlatform.GCP   => GCP
     case WSMCloudPlatform.AZURE => AZURE
+    case WSMCloudPlatform.AWS   => UNKNOWN // AWS exists only in WorkspaceManager at the moment
```

Comment:
This file also deals with CloudPlatform from BPM and Data Repo-- both of those do not yet have AWS, so I chose to not expose AWS here in Rawls.